radixdlt-java/src/main/java/com/radixdlt/client/application/actions/StoreDataAction.java

```diff
@@ -10,15 +10,18 @@
  * An Application Layer Action object which stores data into an address or multiple addresses.
  */
 public class StoreDataAction {
+	private final RadixAddress source;
 	private final Data data;
 	private final List<RadixAddress> addresses;
```

Comment:
Better to replace list with explicit single toAddress since Action class represents the action abstraction.